File: springboot/rest/src/main/java/com/sample/rest/controller/HelloController.java
package com.sample.rest.controller;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
public class HelloController {

    private final Logger logger = LoggerFactory.getLogger(HelloController.class);

    @GetMapping("/hello")
    public String hello() {
        logger.info("THIS IS HELLO LOGGER CLASS !!!");

        System.out.println("THIS IS HELLO Hello Controller ~~ !!!");

        return "hello";
    }

}

File: sqs-demo/src/main/kotlin/be/com/sqsdemo/HelloController.kt
package be.com.sqsdemo

import kotlin.reflect.KClass
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController {
    @GetMapping("/hello")
    fun hello(): String {
        return "Hello World!"
    }
}

class TestClass(
    private val testName: String,
    val publicNameTest: String,
) {
    fun testPublicMethod(): String {
        return "This is a test method !"
    }
}

fun main() {
    val testClass: KClass<TestClass> = TestClass::class


}


